core/constants/src/mill/constants/Util.java

```diff
@@ -10,8 +10,9 @@
 
 public class Util {
 
-  public static boolean isWindows =
-      System.getProperty("os.name").toLowerCase(Locale.ROOT).startsWith("windows");
+  private static String lowerCaseOsName = System.getProperty("os.name").toLowerCase(Locale.ROOT);
```

Comment:
I'd rather not keep this as a field. If we want to be DRY, make it a private method.